src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/ColorFadeT.cs

```diff
@@ -111,7 +111,7 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
             return fValid;
         }
 
-        private bool CheckOtherType(DType otherType, TexlNode otherArg, DType expectedType, IErrorContainer errors, ref Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        private bool CheckOtherType(CheckTypesContext context, DType otherType, TexlNode otherArg, DType expectedType, IErrorContainer errors, ref Dictionary<TexlNode, DType> nodeToCoercedTypeMap, Features features)
```

Comment:
>Features features [](http://example.com/codeflow?start=197&length=17)

This param is not needed if you are already passing context to it.

Reply:
Good point, removed